src/textures/Texture.js

```diff
@@ -94,7 +94,7 @@ class Texture extends EventDispatcher {
 
 	}
 
-	set image( value = null ) {
+	set image( value ) {
```

Comment:
What if a user does this now:
```js
texture.image = undefined;
```
`Source.data` should never be `undefined` but `null`.

Reply:
it is an **accessor** ... not setImage

Reply:
Don't parameter initializers also work in accessors?

Reply:
you're right # TIL setter can have initializer :D